### DISCUSSION TITLE
Implantate und Implantat-Sets in den Samedis.care Katalog integrieren

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#85] [Ideas]
Auch Implantate und entsprechende OP-Sets müssen eingewiesen werden bzw. werden Operateure auf diese von den Herstellern eingewiesen. Diese Einweisungen müssen auch dokumentiert und aufbewahrt werden.

Nähere Informationen hierzu kann ggf. Dr. med. Gebhard von Cossel aus der Sana geben.

**Comment by YvesR** [comment]
Will result into device-model and inventory categories (e.g. `medical-device`, `implant`, `spare parts`, `consumption`, `whatever`).
